internal/provisioner/plain/main.go

```diff
@@ -64,7 +69,7 @@ func main() {
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&systemNamespace, "system-namespace", "rukpak-system", "Configures the namespace that gets used to deploy system resources.")
-	flag.StringVar(&unpackImage, "unpack-image", "quay.io/operator-framework/plain-provisioner:latest", "Configures the container image that gets used to unpack Bundle contents.")
+	flag.StringVar(&unpackImage, "unpack-image", defaultProvisionerImage+":"+defaultProvisionerImageTag, "Configures the container image that gets used to unpack Bundle contents.")
```

Comment:
Does changing the unpacker default image to point to the main tag by default result in always pulling down that image even when running locally? Should we instead be running the locally-built image when using `make run-local`? 

For example, say we are developing the unpacker locally and want to deploy it on the cluster. We would need some way to override this default to point to the `local` tag.  Is that scenario worth introducing a kustomize patch that would set the image to point to the local tag, when `make run-local` is used?